modules/nf-core/tiara/tiara/tests/main.nf.test

```diff
@@ -0,0 +1,42 @@
+
+nextflow_process {
+
+    name "Test Process TIARA_TIARA"
+    script "../main.nf"
+    process "TIARA_TIARA"
+    config "./nextflow.config"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "tiara"
+    tag "tiara/tiara"
+
+    test("test-tiara-tiara") {
```

Comment:
```suggestion
    test("illumina-contigs-fa") {
```

Reply:
@Joon-Klaps , these PRs are semi-automated conversions, you can just commit your suggestions and merge them in.